Polish SDK 4.0 docs - Closes #736 (#737)
LiskHQ/lisk-docs: modules/ROOT/pages/guides/app-development/test-transaction.adoc

}
----

> To test that the `applyAsset()` function is working as expected, write a unit test, like it is done here: {url_github_hello_world_tx_test}[hello_transaction.test.js^]:

Reviewer: ```suggestion
To test that the `applyAsset()` function is working as expected, write a unit test as shown in the example here: {url_github_hello_world_tx_test}[hello_transaction.test.js^]:
```
